### ISSUE TITLE
After installing gorilla on my VS 2015 I can no longer open my xamarin projects

### ISSUE THREAD

**Issue by alpha2aplicativos** [UXDivers/Gorilla-Player-Support#202]
I confess I was thrilled with the app ad and I was a bit frustrated that it did not work properly. I am accustomed to working with WPF and Gorilla seemed to me the alternative of the skies to develop with xamarin however there are many bugs and I can no longer load my projects.
The installation occurred without any problems, however now my visual studio is constantly locking and my device running in genymotion before super fast now is taking a lot to start, even leaving the gorilla player disconnected
![gorilla](https://cloud.githubusercontent.com/assets/22476009/24831303/cada8b5c-1c6d-11e7-8851-a7045e44d0f1.png)

**Comment by LeoHere** [comment]
@alpha2aplicativos sorry for the inconvenience you have experienced. 

Can you send me your Gorilla [Logs](https://github.com/UXDivers/Gorilla-Player-Support/wiki/Gorilla-Player-logs-location)?

Does the problem goes away as soon as you uninstall the addin?

Gorilla VS Addin is pretty lightweight. It only tries to connect to the server once, during VS startup and doing it asynchronously. If the server is not found it will wait until you hit the "Connect..." to try again. The Addin does never tries to connect with the devices, that task is performed by Gorilla Desktop App. I mention this just to clarify a bit how Gorilla works. If you quit the Desktop App Gorilla shouldn't be doing much in background. 

Please also try as follows:
1) Quit VS 2015 and quit Gorilla desktop app
2) Open VS 2015 and open you Xamarin solution (since Gorilla desktop app is down, the addin should not do much)
3) Did you still experience the same issue?

**Comment by alpha2aplicativos** [comment]
Hello Leonardo, thank you for replying. I waited about 10 minutes for my project to load and then this problem stopped. I visualized the logs as you had informed me and did not see any information that could indicate the problem to me. After some reboots of windows the problem stopped and the projects are loading normally and I tested the gorilla with a device in genymotion and it worked well. thanks
